packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -145,6 +144,71 @@ describe('StripeHelper', () => {
     sandbox.restore();
   });
 
+  describe('cachedResult', () => {
+    const expectedResult = { status: 'success' };
+
+    const cacheKey = 'testKey';
+    let refreshFunction;
+
+    beforeEach(() => {
+      refreshFunction = sandbox.stub().returns(expectedResult);
+    });
+
+    it('follows read-through caching logic', async () => {
+      // On first call: cache miss, call refresh function, cache the result
+      let actualResult = await stripeHelper.cachedResult(
+        cacheKey,
+        refreshFunction
+      );
+      assert(mockRedis.get.calledOnce);
+      assert(mockRedis.set.calledOnce);
+      assert(refreshFunction.calledOnce);
+      assert.deepEqual(actualResult, expectedResult);
+
+      refreshFunction.resetHistory();
+      // On subsequent call: cache hit, return cache result
+      actualResult = await stripeHelper.cachedResult(cacheKey, refreshFunction);
+      assert(mockRedis.get.calledTwice);
+      assert(mockRedis.set.calledOnce);
+      assert(refreshFunction.notCalled);
+      assert.deepEqual(actualResult, expectedResult);
+    });
+
+    describe('when there is an error on cache read', () => {
+      it('logs error and returns expected result', async () => {
+        mockRedis.get.restore();
+        sandbox.stub(mockRedis, 'get').rejects(Error);
+
+        const actualResult = await stripeHelper.cachedResult(
+          cacheKey,
+          refreshFunction
+        );
+
+        assert.isTrue(
+          log.error.calledOnceWith('stripeHelper.cachedResult.redis.get.failed')
+        );
+        assert.deepEqual(actualResult, expectedResult);
+      });
+    });
+
+    describe('when there is an error on cache save', () => {
+      it('logs error and returns expected result', async () => {
+        mockRedis.set.restore();
+        sandbox.stub(mockRedis, 'set').rejects(Error);
+
+        const actualResult = await stripeHelper.cachedResult(
+          cacheKey,
+          refreshFunction
+        );
+
+        assert(
+          log.error.calledOnceWith('stripeHelper.cachedResult.redis.set.failed')
+        );
+        assert.deepEqual(actualResult, expectedResult);
+      });
+    });
+  });
```

Comment:
I don't see any tests verifying that redis `get` and `set` were called with the correct arguments.